mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -662,6 +663,27 @@ public void testMultiTableInsert() throws IOException {
     HiveIcebergTestUtils.validateData(target2, target2Records, 1);
   }
 
+  @Test
+  public void testWriteWithDefaultWriteFormat() {
+    Assume.assumeTrue("Tez write is not implemented yet", executionEngine.equals("mr"));
```

Comment:
Do we need to run this for every catalog / fileFormat?
Maybe we do not want to create a test class just for this, but at least skip the unnecessary tests.

Reply:
That's right, it doesn't add any value to run it for different configurations. I'm going to expand the Assume.